src/ReleaseHistory.md

```diff
@@ -13,7 +13,10 @@
 - UER => eliminate unhandled exceptions in rules
 - UEE => eliminate unhandled exceptions in engine
 
-## **v4.0.0-rc4** UNRELEASED
+## **v4.0.0-rc5** UNRELEASED
+
+## **v4.0.0-rc4**
+* DEP: Update Sarif.Sdk submodule from [235394a to ec93dccd](https://github.com/microsoft/sarif-sdk/compare/235394a...ec93dccd). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/ec93dccd/src/ReleaseHistory.md).
```

Comment:
@shaopeng-gh Note that Mike's PR on threading fixes got merged: https://github.com/microsoft/sarif-sdk/pull/2618

So we should update the `sarif-sdk` submodule to reference the appropriate point in `main`.

Reply:
Updated to latest main.